### PR TITLE
Support logging.file.path

### LINKED ISSUE #416: spring boot 2.x logging.path 过期
### Describe the bug

spring boot 2.x版本中，logging.path属性已过期，使用logging.file.path无效

### Environment

- SOFATracer version: latest
- JVM version  1.8
- String boot 2.4.1

### PR DESCRIPTION
**Pull request by glmapper**
FIX #416

### CODE REVIEW

File: tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java [hunk 1]
```diff
@@ -64,6 +64,11 @@
         if (StringUtils.isBlank(loggingRoot)) {
             loggingRoot = System.getProperty("logging.path");
         }
+        // adapter to springboot 2.4.x
+        if (StringUtils.isBlank(loggingRoot)) {
+            loggingRoot = System.getProperty("logging.file.path");
```
Comment (khotyn) [inline tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java:69]: I think `logging.file.path` is not just a system property, but also an SpringBoot property.

**Review by khotyn — CHANGES_REQUESTED** [review]

File: tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java (hunk 1 above)
Comment (glmapper) [reply tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java:69]: > I think `logging.file.path` is not just a system property, but also an SpringBoot property.
yeah, But there is not a change in the tracer, 3.1.0 version adds to the sofa-common-tools, and then through the CommonLoggingConfigurations from SOFABoot tin received all enviroment properties

Comment (khotyn) [reply tracer-core/src/main/java/com/alipay/common/tracer/core/appender/TracerLogRootDaemon.java:69]: OK

**Comment by khotyn** [comment]
@glmapper The Codecov result is not generated in this thread, is there something wrong with our Codecov configuration?

**Review by ujjboy — APPROVED** [review]
LGTM

**Pull request merged**